exercises/rectangles/example.js

```diff
@@ -0,0 +1,38 @@
+class Rectangles {
+
+  static count(diagram) {
+    const rows = diagram.length;
+    const cols = rows !== 0 ? diagram[0].length : 0;
```

Comment:
This could be simply:
```javascript
const cols = rows ? diagram[0].length : 0;
```

If you're concerned about `rows` being a falsy value other than zero, I think a validation check would be appropriate.